addons/mail/static/src/components/chat_window_header/chat_window_header.scss

```diff
@@ -47,29 +30,15 @@
     }
 }
 
-.o_ChatWindowHeader_name {
-    max-height: map-get($sizes, 100);
-}
-
-.o_ChatWindowHeader_rightArea {
-    display: flex;
-    height: map-get($sizes, 100);
-    align-items: center;
-}
-
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
 .o_ChatWindowHeader {
     background-color: $o-brand-odoo;
-    color: $white;
 
     &:not(.o-mobile) {
         cursor: pointer;
```

Comment:
You can use `.o_cursor_pointer` https://github.com/odoo/odoo/blob/master/addons/web/static/src/webclient/webclient.scss#L94

edit: the same for `o_ChatWindowHeader_command`, a few line below